db/revision.go

```diff
@@ -293,7 +293,7 @@ func (db *Database) purgeOldRevisionJSON(docid string, revid string) error {
 //////// UTILITY FUNCTIONS:
 
 func oldRevisionKey(docid string, revid string) string {
-	return fmt.Sprintf("_sync:rev:%s:%d:%s", docid, len(revid), revid)
+	return fmt.Sprintf(base.RevPrefix+"%s:%d:%s", docid, len(revid), revid)
```

Comment:
Concatenation isn't too easy here because of the integer value, so we can keep Sprintf, but remove the additional concatenation.

```suggestion
	return fmt.Sprintf("%s%s:%d:%s", base.RevPrefix, docid, len(revid), revid)
```